modules/deploying-models-using-multiple-gpu-nodes.adoc

```diff
@@ -0,0 +1,266 @@
+:_module-type: PROCEDURE
+
+[id="deploying-models-using-multiple-gpu-nodes_{context}"]
+= Deploying models using multiple GPU nodes
+
+[role='_abstract']
+Deploying models using multiple graphical processing units (GPUs) across multiple nodes can help when deploying large models such as large language models (LLMs).
+
+This procedure shows you how to serve models on {productname-long} across multiple GPU nodes using the vLLM serving framework.
+
+ifndef::upstream[]
+[IMPORTANT]
+====
+*Deploying models using multiple GPU nodes* is currently available in {productname-long} as a Technology Preview feature only. Technology Preview features are not supported with {org-name} production service level agreements (SLAs) and might not be functionally complete. {org-name} does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of {org-name} Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview[Technology Preview Features Support Scope]
+====
+endif::[]
+
+.Prerequisites
+
+* You have cluster administrator privileges for your {openshift-platform} cluster.
+* You have downloaded and installed the {openshift-platform} command-line interface (CLI). See link:https://docs.redhat.com/en/documentation/openshift_container_platform/{ocp-latest-version}/html/cli_tools/openshift-cli-oc#installing-openshift-cli[Installing the OpenShift CLI^].
+
+ifndef::upstream[]
+* You have enabled the operators for your GPU type, such as Node Feature Discovery Operator, NVIDIA GPU Operator, and so on. For more information about enabling accelerators, see link:{rhoaidocshome}{default-format-url}/managing_openshift_ai/working_with_accelerators[Working with accelerators^].
+endif::[]
+ifdef::upstream[]
+* You have enabled the operators for your GPU type, such as Node Feature Discovery Operator, NVIDIA GPU Operator, and so on. For more information about enabling accelerators, see link:{odhdocshome}/working-with-accelerators[Working with accelerators^].
+endif::[]
+
+** You are using one of the following GPU types: `nvidia.com/gpu` (default), `intel.com/gpu`, `amd.com/gpu`, `habana.ai/gaudi`, or you have specified another GPU type in the annotations of the `InferenceService`. 
```

Comment:
only `nvidia.com/gpu` was tested so I think we should remove the rest. @Jooho wdyt?

Reply:
To me, it should work with various gpu type if they are using sole gpu type but I think we can add that the current version only tested with `nvidia.com/gpu`.

Reply:
I think I will remove other GPU types since only Nvidia was tested.